scrapy/extensions/closespider.py

```diff
@@ -32,7 +33,7 @@ def __init__(self, crawler):
         if self.close_on.get('errorcount'):
             crawler.signals.connect(self.error_count, signal=signals.spider_error)
         if self.close_on.get('pagecount'):
-            crawler.signals.connect(self.page_count, signal=signals.response_received)
+            crawler.signals.connect(self.response_received, signal=signals.response_received)
```

Comment:
why did you rename this? (also method declaration below). It does the same, but you think this is more clear? Should you not also change 'pagecount' in
`: if self.close_on.get('pagecount'):`
then?